code/modules/mob/living/carbon/xenomorph/xeno_helpers.dm

```diff
@@ -13,8 +13,18 @@
 	SHOULD_CALL_PARENT(FALSE)
 	return FALSE
 
+/mob/living/carbon/xenomorph/get_crit_threshold()
+	. = ..()
+	if(!endure)
+		return
+	var/datum/action/ability/xeno_action/endure/endure_ability = actions_by_path[/datum/action/ability/xeno_action/endure]
```

Comment:
bloodthirster uses a different path (/endure/nocost) so i don't think this works correctly.

Reply:
good catch this was probably it